packages/webdriverio/src/types.ts

```diff
@@ -217,18 +217,31 @@ type OverwriteCommandFn<
 
 export type CustomLocatorReturnValue = HTMLElement | HTMLElement[] | NodeListOf<HTMLElement>
 
-type Instances = WebdriverIO.Browser | WebdriverIO.MultiRemoteBrowser
+export type Instances = CustomCommands.Instances
+export type CustomCommandOptions<IsElement extends boolean> = CustomCommands.CustomCommandOptions<IsElement>
+export type AddCommandFunction<IsElement extends boolean, T = any, Instance = WebdriverIO.Browser> = IsElement extends true ? AddCommandFnScoped<T | Instance, IsElement> : AddCommandFn
 
 export interface CustomInstanceCommands<T> {
+
     /**
+     * @deprecated use the below overload with options object instead
```

Comment:
The user won't see any other overload docs, only the one the overload applies to
```suggestion
     * @deprecated use option object as 3rd parameter
```